PocketKit/Sources/PocketKit/MyList/SearchItemsList/LocalSavesSearch.swift

```diff
@@ -7,18 +7,13 @@ import Analytics
 
 class LocalSavesSearch {
     private let source: Source
-    private var cache: [String: [PocketItem]] = [:]
 
     init(source: Source) {
         self.source = source
     }
 
     func search(with term: String) -> [PocketItem] {
-        guard cache[term] == nil else {
-            return cache[term] ?? []
-        }
         let items = source.searchSaves(search: term)?.compactMap { PocketItem(item: $0) } ?? []
-        cache[term] = items
-        return cache[term] ?? []
+        return items
```

Comment:
👍 I also added this change to the decision log in the [PRD](https://getpocket.atlassian.net/wiki/spaces/CP/pages/2807660579/PRD+iOS+Search), feel free to edit if I misspoke

![image](https://user-images.githubusercontent.com/6743397/236539489-0f167af6-aa20-470c-b8c3-caf7aa7f48df.png)